packages/wouter/types/router.d.ts

```diff
@@ -5,7 +5,10 @@ import {
   BaseSearchHook,
 } from "./location-hook.js";
 
-export type Parser = (route: Path) => { pattern: RegExp; keys: string[] };
+export type Parser = (
+  route: Path,
+  loose?: boolean
+) => { pattern: RegExp; keys: string[] };
```

Comment:
Would it be possible to add a quick type spec to cover this change? `router.test-d.tsx`